api/v1/type_bootstrap_config.go

```diff
@@ -5,34 +5,77 @@ import "github.com/canonical/k8s-snap-api/internal/util"
 
 	// Seed configuration for the control plane (flat on purpose). Empty values are ignored
+
+	// List of taints to be applied to control plane nodes.
 	ControlPlaneTaints  []string `json:"control-plane-taints,omitempty" yaml:"control-plane-taints,omitempty"`
+	// The CIDR to be used for assigning pod addresses.
+	// If omitted defaults to `10.1.0.0/16`.
 	PodCIDR             *string  `json:"pod-cidr,omitempty" yaml:"pod-cidr,omitempty"`
+	// The CIDR to be used for assigning service addresses.
+	// If omitted defaults to `10.152.183.0/24`.
 	ServiceCIDR         *string  `json:"service-cidr,omitempty" yaml:"service-cidr,omitempty"`
+	// Determines if RBAC should be disabled.
+	// If omitted defaults to `false`.
 	DisableRBAC         *bool    `json:"disable-rbac,omitempty" yaml:"disable-rbac,omitempty"`
+	// The port number for kube-apiserver to use.
+	// If omitted defaults to `6443`.
 	SecurePort          *int     `json:"secure-port,omitempty" yaml:"secure-port,omitempty"`
+	// The port number for k8s-dqlite to use.
+	// If omitted defaults to `9000`.
 	K8sDqlitePort       *int     `json:"k8s-dqlite-port,omitempty" yaml:"k8s-dqlite-port,omitempty"`
+	// The type of datastore to be used.
+	// If omitted defaults to `k8s-dqlite`.
+	//
+	// Can be used to point to an external datastore like etcd.
+	//
+	// Possible Values: `k8s-dqlite | external`.
 	DatastoreType       *string  `json:"datastore-type,omitempty" yaml:"datastore-type,omitempty"`
+	// The server addresses to be used when `datastore-type` is set to `external`.
 	DatastoreServers    []string `json:"datastore-servers,omitempty" yaml:"datastore-servers,omitempty"`
+	// The CA certificate to be used when communicating with the external datastore.
 	DatastoreCACert     *string  `json:"datastore-ca-crt,omitempty" yaml:"datastore-ca-crt,omitempty"`
+	// The client certificate to be used when communicating with the external
+	// datastore.
 	DatastoreClientCert *string  `json:"datastore-client-crt,omitempty" yaml:"datastore-client-crt,omitempty"`
+	// The client key to be used when communicating with the external datastore.
 	DatastoreClientKey  *string  `json:"datastore-client-key,omitempty" yaml:"datastore-client-key,omitempty"`
 
 	// Seed configuration for certificates
+
+	// List of extra SANs to be added to certificates.
 	ExtraSANs []string `json:"extra-sans,omitempty" yaml:"extra-sans,omitempty"`
 
 	// Seed configuration for external certificates (cluster-wide)
+
+	// The CA certificate to be used for Kubernetes services.
+	// If omitted defaults to an auto generated certificate.
 	CACert                          *string `json:"ca-crt,omitempty" yaml:"ca-crt,omitempty"`
+	// The CA key to be used for Kubernetes services.
+	// If omitted defaults to an auto generated key.
 	CAKey                           *string `json:"ca-key,omitempty" yaml:"ca-key,omitempty"`
 	ClientCACert                    *string `json:"client-ca-crt,omitempty" yaml:"client-ca-crt,omitempty"`
```

Comment:
Any reason that you do not set comments here and below?

Reply:
I've only moved what was already here: https://github.com/canonical/k8s-snap/blob/ee017318d958b4c7ed1ab78d3cf4d8b7e94559c5/docs/src/snap/reference/bootstrap-config-reference.md